.github/workflows/build.yml

```diff
@@ -114,62 +114,11 @@ jobs:
       with:
         go-version: '1.21'
 
-    - name: Build pgroll (Linux amd64)
-      run: CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.linux.amd64
-
-    - name: Build pgroll (Linux arm64)
-      run: CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.linux.arm64
-
-    - name: Build pgroll (MacOS amd64)
-      run: CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.macos.amd64
-
-    - name: Build pgroll (MacOS arm64)
-      run: CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.macos.arm64
-
-    - name: Build pgroll (Windows)
-      run: CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.win.amd64
-
-    - uses: actions/upload-artifact@v3
-      with:
-        name: pgroll.linux.amd64
-        path: pgroll.linux.amd64
-
-    - uses: actions/upload-artifact@v3
-      with:
-        name: pgroll.linux.arm64
-        path: pgroll.linux.arm64
-
-    - uses: actions/upload-artifact@v3
-      with:
-        name: pgroll.macos.amd64
-        path: pgroll.macos.amd64
-
-    - uses: actions/upload-artifact@v3
-      with:
-        name: pgroll.macos.arm64
-        path: pgroll.macos.arm64
-
-    - uses: actions/upload-artifact@v3
+    - name: Run GoReleaser
+      uses: goreleaser/goreleaser-action@v3
```

Comment:
The latest version of the action is `v5` [[here](https://github.com/goreleaser/goreleaser-action)]. Is there a particular reason why we don't use the latest?

Reply:
will use v5 now